src/Ext/HouseType/Body.h

```diff
@@ -0,0 +1,45 @@
+#pragma once
+#include <HouseTypeClass.h>
+
+#include <Helpers/Macro.h>
+#include <Utilities/Container.h>
+#include <Utilities/TemplateDef.h>
+
+class HouseTypeExt
+{
+public:
+	using base_type = HouseTypeClass;
+	class ExtData final : public Extension<HouseTypeClass>
+	{
+	public:
+		Nullable<bool> CountryCrew;
```

Comment:
```suggestion
		Valueable<bool> CountryCrew;
```
No need to use Nullable if you don't have another default value here. Doubt if this is necessary again